CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v1.34.0
+------------------------------
+*March 05, 2019*
```

Comment:
Super tiny thing – can just make this `5` rather than `05` as that's consistent with the rest of the single digit dates in the changelog